src/configure

```diff
@@ -84,6 +100,9 @@ static_fst=false
 use_cuda=true
 dynamic_kaldi=false
 mkl_threading=sequential
+HOST=""
+TARGET_ARCH=""
```

Comment:
Can you please add a comment above these, saying something like the following (edited for accuracy):
```
# HOST and TARGET_ARCH are only set on the command line in case of cross-compilation.
# If HOST is not set, TARGET_ARCH will be set to `uname -m` inside the script.
```